mailheader.js

```diff
@@ -98,7 +98,7 @@ class Header {
     }
 
     get_all (key) {
-        return this.headers[key.toLowerCase()] || [];
+        return [].concat(this.headers[key.toLowerCase()] || []);
```

Comment:
Instead of Array.concat, how about using the es6 spread operator?

```suggestion
        return Object.freeze([...(this.headers[key.toLowerCase()] || [])]);
```

Reply:
yes, but this code base is old I was not sure it would be accepted this way ;-) perfect